contrib/database/sql/conn.go

```diff
@@ -133,6 +133,25 @@ type traceParams struct {
 	meta       map[string]string
 }
 
+type contextKey int
+
+const spanTagsKey contextKey = 0 // map[string]string
+
+// WithSpanTags is used if you want to set the span tag in a context.
+// You can set span tag in the context like this
+//
+// ctx = WithSpanTags(ctx, map[string]string{"tag_nane":"tag_value"})
```

Comment:
I think the example is not needed, but if you do wish to keep it, let's make sure the following points are met:

* It is correctly formatted using `gofmt`
* There are exactly 3 spaces between the comment start (`//`) and the first character of the line of code to ensure it displays _as code_ in `godoc`
* There are no `...` and it's fully working code.

It'd be nice to either remove it, or correct it. Up to you!

Reply:
Removed it!!